ui/imports/shared/views/HistoryView.qml

```diff
@@ -136,29 +136,47 @@ ColumnLayout {
         ScrollBar.vertical: StatusScrollBar {}
 
         section.property: "date"
-        section.delegate: Item {
+        section.delegate: ColumnLayout {
             id: sectionDelegate
 
             readonly property bool isFirstSection: ListView.view.firstSection === section
 
             width: ListView.view.width
-            height: isFirstSection || section.length > 1 ? 40 : 0 // 1 because we don't use empty for loading state
+            height: isFirstSection ? 38 : (section.length > 1 ? 58 : 0) // 1 because we don't use empty for loading state
             visible: height > 0 // display always first section. Other sections when more items are being fetched must not be visible
+            spacing: 0
 
             required property string section
 
+            Item {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 20
+                visible: !sectionDelegate.isFirstSection
+            }
+
+            Rectangle {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 1
+                color: Theme.palette.baseColor2
+            }
+
             StatusTextWithLoadingState {
                 id: sectionText
 
-                anchors.verticalCenter: parent.verticalCenter
+                Layout.fillWidth: true
+                Layout.leftMargin: 16
+                Layout.fillHeight: true
+                Layout.bottomMargin: 8
```

Comment:
The only prop you need here is `Layout.preferedWidth: parent.width`

Reply:
Sorry @saledjenic but I don't understand you point here. Text in Figma has specific left padding of 16 pixels also it needs proper vertical position, that's why those properties are here.

Could you explain your idea a little more, please?

Reply:
I wanted to say that this component always knows for its implicit size and it participates in `ColumnLayout` not in `Item` any more, so the things should be set according to that, but that requires some more changes as well (changing a bit qml structure to achieve the same UI effect).

Reply:
Updated. Now using only top margin and paddings to position text.